server/core/src/main/java/dev/slimevr/tracking/trackers/TrackerResetsHandler.kt

```diff
@@ -432,6 +453,28 @@ class TrackerResetsHandler(val tracker: Tracker) {
 		return totalMatrix.toQuaternion()
 	}
 
+	/**
+	 * Update the yaw reset smoothing time 
+	 */
+	@Synchronized
+	fun update() {
+		if (yawResetSmoothTimeRemain != 0.0f) {
+			var deltaTime = 0.001f
+			if (::fpsTimer.isInitialized) {
+				deltaTime = fpsTimer.timePerFrame
+			}
+			yawResetSmoothTimeRemain = max(0.0f, yawResetSmoothTimeRemain - deltaTime)
+			if (yawResetSmoothTimeRemain == 0.0f) {
```

Comment:
Can simplify this code and avoid some precision-based bugs
```suggestion
		if (yawResetSmoothTimeRemain > 0.0f) {
			var deltaTime = 0.001f
			if (::fpsTimer.isInitialized) {
				deltaTime = fpsTimer.timePerFrame
			}
			yawResetSmoothTimeRemain = yawResetSmoothTimeRemain - deltaTime
			if (yawResetSmoothTimeRemain <= 0.0f) {
```

Reply:
I'm more with having an accurate 0.0f as some sort of flag and assigning 0.0f would always be precise. Anyway, the sign could still be a boolean-ish flag so probably it's fine.